File: src/store/DTO/Preference.ts
export class DTOPreferenceReq {
  firstName: string;
  lastName: string;
  DNI: string;
  phoneNumber: string;
  mail: string;

  constructor(
    firstName: string,
    lastName: string,
    DNI: string,
    phoneNumber: string,
    mail: string,
  ) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.DNI = DNI;
    this.phoneNumber = phoneNumber;
    this.mail = mail;
  }
}

File: DTO/Preference.cs
using System.ComponentModel.DataAnnotations;
using MercadoPago.Client.Payment;
using Newtonsoft.Json;

namespace DTO;



public class DTOPreferenceReq
{
  [Required]
  public string firstName { get; set; }
  [Required]
  public string lastName { get; set; }
  [Required]
  public string DNI { get; set; }
  [Required]
  public string phoneNumber { get; set; }
  [Required]
  public string mail { get; set; }

}



